user_id.py

```diff
@@ -0,0 +1,43 @@
+"""Use a given name string, a family name string, and a random integer
+between 100 and 999 to create a user_id.
+"""
+import random
+import re
+import name
+
+def is_valid(user_id_str):
+    """Take in a string containing a user ID. Return True if it is
+    valid user ID. Otherwise, return False.
+    """
+    if re.search("[a-zA-Z]+[0-9][0-9][0-9]", user_id_str):
+        return True
+    return False
+
+def create(given_name, family_name):
```

Comment:
i'm not sure why we're adding a random integer here.

let's use an integer as a user id - that's pretty typical - rather than a string with a random number. you have two options, you can use 

http://stackoverflow.com/questions/2817038/how-can-i-create-a-sequence-in-sqlite

or you can use the hashlib i showed you earlier, that will map any strings (or anything really) to a single number. in python's case, it creates another string that is the hex value (remember, binary, octal, hex, decimal, etc...) of that string's number because it's usually smaller. for example:

```
import hashlib
m = hashlib.md5()
m.update("coit")
m.update("scharringhausen")
print m.hexdigest()
```

This is technically representing a number, but python stores it as a string. If you'd like, you can keep your user id a string *if you go with the hex value*, or an integer *if you go with the sequence id*. 

Whatever you go with, i'd like a simple script that given a first and last name will get me the user id of that.

The benefit of the hex value is I can run that script on a first and last name not in the test set and still get a user id!